### PR TITLE
chore: update send command consistent

### PR DESCRIPTION
**Pull request by jowparks**
## Summary
Updates references to `account` to say `wallet`
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/wallet/send.ts [hunk 1]
```diff
@@ -11,16 +11,16 @@ export class Send extends IronfishCommand {
...
-    '$ ironfish accounts:send -a 2 -o 0.00000001 -t 997c586852d1b12da499bcff53595ba37d04e4909dbdb1a75f3bfd90dd7212217a1c2c0da652d187fc52ed -f otheraccount -m my_message_for_the_transaction',
+    '$ ironfish wallet:send -a 2 -o 0.00000001 -t 997c586852d1b12da499bcff53595ba37d04e4909dbdb1a75f3bfd90dd7212217a1c2c0da652d187fc52ed',
+    '$ ironfish wallet:send -a 2 -o 0.00000001 -t 997c586852d1b12da499bcff53595ba37d04e4909dbdb1a75f3bfd90dd7212217a1c2c0da652d187fc52ed -f otherwallet',
+    '$ ironfish wallet:send -a 2 -o 0.00000001 -t 997c586852d1b12da499bcff53595ba37d04e4909dbdb1a75f3bfd90dd7212217a1c2c0da652d187fc52ed -f otherwallet -m my_message_for_the_transaction',
   ]
 
   static flags = {
     ...RemoteFlags,
-    account: Flags.string({
+    wallet: Flags.string({
```
Comment (rohanjadvani) [inline ironfish-cli/src/commands/wallet/send.ts:21]: This should still be account. `-f` refers to the account under the wallet funds are being sent from

File: ironfish-cli/src/commands/wallet/send.ts [hunk 2]
```diff
@@ -107,8 +107,8 @@ export class Send extends IronfishCommand {
 
       if (!defaultAccount) {
         this.error(
-          `No account is currently active.
-           Use ironfish accounts:create <name> to first create an account`,
+          `No wallet is currently active.
+           Use ironfish wallet:create <name> to first create a wallet`,
```
Comment (rohanjadvani) [inline ironfish-cli/src/commands/wallet/send.ts:111]:
```suggestion
           Use ironfish wallet:create <name> to first create an account`,
```

File: ironfish-cli/src/commands/wallet/send.ts [hunk 3]
```diff
@@ -64,7 +64,7 @@ export class Send extends IronfishCommand {
     let amount = null
     let fee = null
     let to = flags.to?.trim()
-    let from = flags.account?.trim()
+    let from = flags.wallet?.trim()
```
Comment (rohanjadvani) [inline ironfish-cli/src/commands/wallet/send.ts:67]:
```suggestion
    let from = flags.account?.trim()
```

**Review by rohanjadvani — CHANGES_REQUESTED** [review]

File: ironfish-cli/src/commands/wallet/send.ts (hunk 1 above)
Comment (NullSoldier) [reply ironfish-cli/src/commands/wallet/send.ts:21]: This should be account not wallet

File: ironfish-cli/src/commands/wallet/send.ts [hunk 4]
```diff
@@ -11,16 +11,16 @@ export class Send extends IronfishCommand {
...
+    '$ ironfish wallet:send -a 2 -o 0.00000001 -t 997c586852d1b12da499bcff53595ba37d04e4909dbdb1a75f3bfd90dd7212217a1c2c0da652d187fc52ed -f otherwallet -m my_message_for_the_transaction',
   ]
 
   static flags = {
     ...RemoteFlags,
-    account: Flags.string({
+    wallet: Flags.string({
       char: 'f',
-      description: 'The account to send money from',
+      description: 'The wallet to send money from',
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/wallet/send.ts:23]: not a wallet

**Review by rohanjadvani — APPROVED** [review]

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**